secrets/farallon.yaml

```diff
@@ -0,0 +1,33 @@
+sops:
+    kms: []
+    gcp_kms:
+        - resource_id: projects/two-eye-two-see/locations/global/keyRings/sops-keys/cryptoKeys/similar-hubs
+          created_at: "2021-05-03T18:24:30Z"
+          enc: CiQA4OM7eBvDdmSosNwoV1H4lGprhSnF6VzDyRaW6U1D9qi6XHQSSQBy9hCY8tILK+/MkhaLO9jCz3fDyhOiF1lxq+zw7GZF8USAZ1ihXFp29CLO69PQa5ejJqZffdWydRTJxuQ1Gd+pEHzC2KmCrGc=
+    azure_kv: []
+    hc_vault: []
+    age: []
+    lastmodified: "2021-05-03T18:24:31Z"
+    mac: ENC[AES256_GCM,data:lSuWapMSsXuAeeSCKHGRsVRXtmeUpCQVTTKDFYHfFADyWF+t8TgwNvRvIuicK69esHW8GnRb5saakKzmlZjwFJKZ+lU8ADXRIY38uRo/4OEvzKhi71lNHxSqZHG/RIPbFsuPoF+xNpqBmfoGmKhrJTxP7zs5hKz2zwZ5GuUdgec=,iv:yVgiI7pbLWPXl1++ZVhg31ASIZn7FgjU8NbouldqC5s=,tag:HsQXJX5JbLVuB10c4JzMiA==,type:str]
+    pgp: []
+    unencrypted_suffix: _unencrypted
+    version: 3.7.1
```

Comment:
In the related issue you suggested:

> Yep! For kops I was thinking we can just ship the kubeconfig generated by KUBECONFIG=secrets/farallon.yaml kops export kubecfg farallon-2i2c.k8s.local --admin but there are probably other options too

When I run that command I get a file in `secrets/farallon.yaml`, but that one seems unencrypted... I guess I am missing some sops stuff.

Reply:
Yeah, if you run `sops -i -e <path>` it will encrypt it in place